src/js/contentScript.js

```diff
@@ -1,11 +1,45 @@
 (function() {
-	console.log('content script loaded');
-	
-	const usdRegex = /\$( ?[0-9]+)/g;
-	const usdToRupeeRatio = 69.58;
-
-	document.body.innerHTML = document.body.innerHTML.replace(usdRegex, (_, val) => {
-		const numval = Number(val);
-		return Number.isNaN(numval) ? `$ ${val}` : `₹ ${usdToRupeeRatio * numval}`;
+	function replace($node, regex, replaceMethod) {
+		const queue = [$node];
+		while (queue.length) {
```

Comment:
Are we doing this, so that headlines of search results get converted first, and then the descriptions of all results?